include/zephyr/bluetooth/classic/device_id.h

```diff
@@ -0,0 +1,42 @@
+/** @file
+ *  @brief Device Identification Protocol handling.
+ */
+
+/*
+ * Copyright 2024 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_DEVICE_ID_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_DEVICE_ID_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Device Identification Structure. */
+struct device_id_info {
```

Comment:
got it， next patch would fix it

Reply:
Please amend the commit itself which introduces an issue instead of fixing it in a later commit (in the same PR). With a single commit you can just use `git commit --amend` and with multiple ones `git rebase -i` is useful. Subsequently you'll need to use `git push --force` to update the PR.

Reply:
> Please amend the commit itself which introduces an issue instead of fixing it in a later commit (in the same PR). With a single commit you can just use `git commit --amend` and with multiple ones `git rebase -i` is useful. Subsequently you'll need to use `git push --force` to update the PR.

done